src/apigateway/activation.ts

```diff
@@ -19,14 +20,8 @@ export async function activate(activateArguments: {
     const extensionContext = activateArguments.extContext.extensionContext
     const regionProvider = activateArguments.extContext.regionProvider
     extensionContext.subscriptions.push(
-        vscode.commands.registerCommand(
-            'aws.apig.copyUrl',
-            async (node: RestApiNode) => await copyUrlCommand(node, regionProvider)
-        )
-    )
-
-    extensionContext.subscriptions.push(
-        vscode.commands.registerCommand(
+        Commands.register('aws.apig.copyUrl', async (node: RestApiNode) => await copyUrlCommand(node, regionProvider)),
+        Commands.register(
```

Comment:
formatting is from prettier. This whole PR is just search-and-replace except for the `autoconnect` cases.